apps/full-site-editing/full-site-editing-plugin/editor-site-launch/src/launch-steps/domain-step/index.tsx

```diff
@@ -0,0 +1,46 @@
+/**
+ * External dependencies
+ */
+import * as React from 'react';
+import { Button } from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import LaunchStep, { Props as LaunchStepProps } from '../../launch-step';
+import './styles.scss';
+
+const DomainStep: React.FunctionComponent< LaunchStepProps > = ( { onPrevStep, onNextStep } ) => {
+	const handleBack = () => {
+		onPrevStep && onPrevStep();
+	};
+
+	const handleContinue = () => {
+		onNextStep && onNextStep();
+	};
+
+	return (
+		<LaunchStep className="domain-step">
+			<div className="launch-step__header">
+				<div className="launch-step__heading">
+					<h1 className="launch-step__title">{ __( 'Choose a domain', 'full-site-editing' ) }</h1>
+					<p className="launch-step__subtitle">
+						{ __( 'Free for the first year with any paid plan', 'full-site-editing' ) }
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **17** times:
`translate( 'Free for the first year with any paid plan.' )` *ES Score: 11*
See 1 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/plans-grid-fse-smart-component)